src/patchAnchorElements.js

```diff
@@ -6,11 +6,14 @@ const BLOCK_NODE_NAME_PATTERN = /(P|H1|H2|H3|H4|H5|H6)/;
 
 export default function patchAnchorElements(doc: Document): void {
   Array.from(doc.querySelectorAll('a[id]')).forEach(patchAnchorElement);
+  Array.from(
+    doc.querySelectorAll('h1[id], h2[id], h3[id], h4[id], h5[id], h6[id]')
+  ).forEach(patchHeaderElements);
 }
 
 function patchAnchorElement(node: HTMLElement): void {
   const {id} = node;
-  if (id && node.childElementCount === 0) {
+  if (id && id.indexOf('t.') !== 0 && node.childElementCount === 0) {
```

Comment:
Could you please document what does `id.indexOf('t.')` mean?
GoogleDoc is like a blackbox and we're making assumptions that developers may not be aware of.